packages/babel-generator/test/fixtures/escapes/options.json

```diff
@@ -1,4 +1,6 @@
 {
```

Comment:
ditto, the `numbers: "hexadecimal"` should be moved to `options.json` _inside_ `escapes/numeric-literals`.